Code/Tools/ProjectManager/Source/ScreensCtrl.cpp

```diff
@@ -33,6 +33,8 @@ namespace O3DE::ProjectManager
         m_tabWidget->tabBar()->setFocusPolicy(Qt::TabFocus);
         m_screenStack->addWidget(m_tabWidget);
         connect(m_tabWidget, &QTabWidget::currentChanged, this, &ScreensCtrl::TabChanged);
+
+        m_downloadController = downloadController;
```

Comment:
same thing move this into the constructor as arguments.

Reply:
you probably want to delete this line now.

Reply:
Good catch, fixed!